pkg/kubectl/describe.go

```diff
@@ -889,6 +899,38 @@ func describeNode(node *api.Node, pods []*api.Pod, events *api.EventList) (strin
 	})
 }
 
+func getNodeTotalLimits(node *api.Node, pods []*api.Pod) (map[api.ResourceName]resource.Quantity, error) {
+	limits := map[api.ResourceName]resource.Quantity{}
+	for _, pod := range pods {
+		limit, err := getPodTotalLimits(pod)
+		if err != nil {
+			return nil, err
+		}
+		for podLimitName, podLimitValue := range limit {
+			if value, ok := limits[podLimitName]; !ok {
+				limits[podLimitName] = podLimitValue
+			} else if _, err = value.Add(podLimitValue); err != nil {
+				return nil, err
+			}
+		}
+	}
+	return limits, nil
+}
+
+func getPodTotalLimits(pod *api.Pod) (map[api.ResourceName]resource.Quantity, error) {
+	limits := map[api.ResourceName]resource.Quantity{}
+	for _, container := range pod.Spec.Containers {
+		for name, quantity := range container.Resources.Limits {
```

Comment:
Once #11713 goes in, the fit check will use  `container.Resources.Request` rather than `container.Resources.Limit`.  So this will have to change.  Don't care which goes in first: this or #11713, but whichever goes in second should change this.  

@AnanyaKumar